README.md

```diff
@@ -345,9 +347,10 @@ The default `shouldTrackOutboundForm` option will consider a form submission fro
 ga('require', 'autotrack', {
   shouldTrackOutboundForm: function(form) {
     var action = form.getAttribute('action');
-    // Checks that the action starts with "http" to exclude relative paths,
-    // then checks that the action does not contain the string "example.com".
-    return action.indexOf('http') === 0 && action.indexOf('example.com') < 0;
+    // Checks that the action is set and starts with "http" to exclude relative // paths, then checks that it does not contain the string "example.com".
```

Comment:
can you do multi-line comment like before so users can read without having to scroll right